Adds Port Configuration for Validators (#1325)
maticnetwork/matic-docs: docs/maintain/validate/additional-info/known-issues.md

import useBaseUrl from '@docusaurus/useBaseUrl';

:::info

> If you have any queries about the Validator process, as well as issues with Heimdall and Bor, please head to the <ins>[Validator FAQ](/docs/faq/validator-faq.md)</ins>.

Reviewer: ```suggestion
If you have any queries about the Validator process or facing issues with Heimdall and Bor, head over to the [<ins>Validator FAQ</ins>](/faq/validator-faq.md) page.
```
